tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```
@@ -1615,11 +1615,12 @@ everflow/test_everflow_per_interface.py::test_everflow_per_interface[ipv6-erspan
 
 everflow/test_everflow_per_interface.py::test_everflow_per_interface[ipv6-erspan_ipv6-default]:
   skip:
-    reason: "SAI_STATUS_NOT_SUPPORTED for everflow over IPv6 on Arista-7260CX3 and Arista-7060CX"
-    conditions_logical_operator: and
+    reason: "SAI_STATUS_NOT_SUPPORTED for everflow over IPv6 on Arista-7260CX3 and Arista-7060CX. Skip everflow per interface IPv6 test on unsupported platforms"
+    conditions_logical_operator: or
     conditions:
       - "https://github.com/sonic-net/sonic-mgmt/issues/19096"
       - "platform in ['x86_64-arista_7260cx3_64', 'x86_64-arista_7060_cx32s']"
+      - "asic_type in ['mellanox']"
```

I think we shouldn't skip it to all platform of mellanox, it's better to specify the platform `x86_64-nvidia_sn5640-r0`


```suggestion
- "platform in ['x86_64-arista_7260cx3_64', 'x86_64-arista_7060_cx32s', 'x86_64-nvidia_sn5640-r0']"
```